editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -787,26 +797,69 @@ export function createWrapInGroupActions(
   const elementPropsOptic = fromTypeGuard<JSXElementLike, JSXElement>(isJSXElement).compose(
     fromField('props'),
   )
-  const childComponents: Array<{ element: JSXElementLike; metadata: ElementInstanceMetadata }> =
-    orderedActionTargets.map((p) => {
+
+  function removeMarginsFromGroupChildElement(
+    element: GroupChildElement,
+    elementMetadata: ElementInstanceMetadata,
+  ) {
+    switch (element.type) {
+      case 'JSX_ELEMENT':
+      case 'JSX_FRAGMENT':
+        // Remove any margin properties from the child.
+        const elementLike = modify(elementPropsOptic, removeMarginProperties, element)
+        return {
+          element: elementLike,
+          metadata: elementMetadata,
+        }
+
+      case 'JSX_CONDITIONAL_EXPRESSION':
+        // Remove any margin properties from the branches.
+        let conditional: JSXConditionalExpression = { ...element }
+        if (isJSXElementLike(conditional.whenTrue)) {
+          conditional.whenTrue = modify(
+            elementPropsOptic,
```

Comment:
If you change this line like this you wont need the containing if statement.
```suggestion
            fromField('whenTrue').compose(elementPropsOptic),
```